exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -166,9 +165,7 @@ private static TransactionResult getTransactionResult(
     @SerializedName("success")
     SUCCESS,
     @SerializedName("error")
-    ERROR,
-    @SerializedName("panic")
-    PANIC
+    ERROR
```

Comment:
That doesn't look like a complete definition of possible errors:

If I understand correctly, they are serialized using this enum: 
https://github.com/exonum/exonum/blob/86379cdee673dfda94b6b1e23ddedb5d512664a7/exonum/src/runtime/error.rs#L378-L384 

and this mapping
https://github.com/exonum/exonum/blob/86379cdee673dfda94b6b1e23ddedb5d512664a7/exonum/src/runtime/error.rs#L386

Reply:
Support for this may be done in a separate PR — but in that case please document that in Jira properly.